packages/wallets/src/components/Base/IconButton/IconButton.tsx

```diff
@@ -26,7 +27,7 @@ const IconButton = forwardRef<HTMLButtonElement, IconButtonProps>(
 
         return (
             <button className={iconButtonClassNames} disabled={disabled} onClick={onClick} ref={ref} {...rest}>
-                <div className='wallets-icon-button__icon'>{icon}</div>
+                <div className={`wallets-icon-button__icon wallets-icon-button__icon-size--${iconSize}`}>{icon}</div>
```

Comment:
right now that div have two "block" classes: 
wallets-icon-button__icon and wallets-icon-button__icon-size
(__icon and __icon-size - the second one does not sound like a block)

while it should have one, e.g.
.wallets-icon-button__icon--xl

Reply:
Updated